campaigns/mofo-eoy-2015/snippet.html

```diff
@@ -0,0 +1,459 @@
+			      c-0.1-0.5,1-1,3.1-1.8c0-1.4,0-2.6-0.2-3.6c-0.1-0.4,0.1-0.7,0.5-0.8c0.4-0.1,0.9,0.1,1.6,0.4s1,0.7,1.1,1.2
+			      c0.1,0.4,0.1,1.1,0,1.9c0.7-0.2,1.2-0.3,1.6-0.3c1.1-0.2,1.7,0,1.8,0.4c0.1,0.3-0.2,0.6-0.8,1c-0.6,0.3-1.5,0.7-2.8,1.2
+			      C118.4,28.1,118.3,28.7,118.2,29.4z"/>
+		      <path d="M131.2,28.9c-0.9,0.7-1.8,1.2-2.8,1.4c-1,0.2-1.9,0.3-2.5,0.2c-0.1,0.4-0.1,0.8,0,1.2s0.2,0.7,0.3,0.8
+			      c0.2,0.1,0.5,0.2,1,0.1c1.1-0.2,2-0.5,2.7-0.8c0.7-0.3,1.1-0.5,1.1-0.5c0.6-0.1,0.9,0,1,0.4c0.1,0.6-0.3,1.2-1.1,1.9
+			      c-0.8,0.7-1.9,1.1-3.1,1.3s-2.3,0-3.1-0.6c-0.9-0.6-1.4-1.5-1.6-2.5c-0.2-1-0.1-2.2,0.3-3.4c0.4-1.2,1-2.3,1.8-3.3
+			      c0.8-0.9,1.6-1.5,2.6-1.7s1.8,0.1,2.8,0.9s1.5,1.6,1.6,2.5C132.4,27.5,132.1,28.2,131.2,28.9z M129.2,26.3
+			      c-0.4-0.2-0.7-0.3-0.9-0.3c-0.3,0-0.6,0.3-0.9,0.8c-0.4,0.5-0.7,1-0.9,1.7c0.1,0,0.3,0,0.6-0.1c2-0.4,2.9-0.9,2.8-1.6
+			      C129.7,26.7,129.5,26.5,129.2,26.3z"/>
+		      <path d="M134.1,26.1l-0.6-2.4c-0.1-0.8,0-1.3,0.5-1.3c0.3-0.1,0.6,0,0.9,0.2c0.3,0.2,0.6,0.5,0.8,0.9c0.4-0.5,0.9-1,1.4-1.3
+			      c0.5-0.4,1-0.6,1.5-0.7c1.1-0.2,1.6,0,1.7,0.5c0,0.3,0,0.5,0,0.6c-0.1,0.1-0.1,0.2-0.2,0.3c0,0.1-0.1,0.2-0.2,0.3
+			      c-0.1,0.1-0.2,0.2-0.2,0.3c-0.3,0.3-0.5,0.5-0.6,0.6c-0.6,0.5-1,1.1-1.4,1.8c-0.4,0.7-0.6,1.5-0.6,2.6c-0.1,1.7-0.3,3-0.5,3.9
+			      c-0.2,0.9-0.6,1.4-1,1.4c-0.5,0.1-0.8-0.1-1.1-0.5c-0.3-0.4-0.5-0.8-0.6-1.2c-0.1-0.4,0-1,0.1-1.8c0.1-0.8,0.2-1.5,0.2-2.1
+			      C134.2,27.4,134.2,26.8,134.1,26.1z"/>
+		      <path d="M142.8,14.8c-0.1-0.4,0.1-0.7,0.6-0.7c0.4-0.1,1.1,0.1,2,0.6c0.9,0.5,1.3,0.9,1.4,1.2c0.2,1,0.2,3,0,6.1
+			      c-0.1,3.1-0.5,4.7-0.9,4.8c-0.4,0.1-0.8-0.1-1.3-0.4c-0.4-0.3-0.7-0.7-0.8-1.2c-0.1-0.2-0.1-0.6-0.1-1.4c0-0.8,0-2-0.2-3.7
+
+			      C143.3,18.4,143.1,16.7,142.8,14.8z M143.2,30.2c-0.1-0.4,0-0.8,0.3-1.2c0.3-0.4,0.6-0.7,0.9-0.8c0.3-0.1,0.8,0.3,1.4,1
+			      c0.6,0.7,0.9,1.2,1,1.6c0,0.2,0,0.4-0.2,0.7c-0.2,0.3-0.4,0.4-0.6,0.5c-0.4,0.1-1-0.1-1.6-0.5C143.6,31,143.3,30.6,143.2,30.2z"/>
+	      </g>
+      </g>
+      </svg>
+    </div>
+    <div class="donation-amounts">
+      <button class="button donation-amount-first" data-amount="{{ donationAmountFirst|safe }}">{{ donationAmountFirst|safe }} {{ currencySymbol|safe }}</button>
+      <button class="button donation-amount-second" data-amount="{{ donationAmountSecond|safe }}">{{ donationAmountSecond|safe }} {{ currencySymbol|safe }}</button>
+      <button class="button donation-amount-third" data-amount="{{ donationAmountThird|safe }}">{{ donationAmountThird|safe }} {{ currencySymbol|safe }}</button>
+      <button class="button donation-amount-fourth" data-amount="{{ donationAmountFourth|safe }}">{{ donationAmountFourth|safe }} {{ currencySymbol|safe }}</button>
+
+      <a class="button button-red button-wide donate-button  {{ buttonAnimate }}" target="_blank" href="#">{{ donateButtonText|safe }}</a>
```

Comment:
Hm, so after looking at https://mozilla.github.io/nunjucks/templating.html#autoescaping

It's my interpretation than only `<p>{{ text|safe }}</p>` needs safe, the rest are better without it.